### PR TITLE
Adding CONTRIBUTING.md and update version number

### PR DESCRIPTION
**Pull request by conniey**
* Update version number to 5.0.0-preview.1
* Add contributing.md

### CODE REVIEW

File: eventhubs/client/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+[opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Getting started
+
+Before working on a contribution, it would be beneficial to familiarize yourself with the process and guidelines used
+for the Azure SDKs so that your submission is consistent with the project standards and is ready to be accepted with
+fewer changes requested. In particular, it is recommended to review:
+
+- [Azure SDK README][github-general], to learn more about the overall project and processes used.
+- [Azure SDK Design Guidelines][design-guidelines], tounderstand the general guidelines for the SDKs across all
```
Comment (JonathanGiles) [inline eventhubs/client/CONTRIBUTING.md]:
```suggestion
- [Azure SDK Design Guidelines][design-guidelines], to understand the general guidelines for the SDKs across all
```

File: eventhubs/client/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- Azure Event Hubs instance
+    - Step-by-step guide for [creating an Event Hub using the Azure Portal][event_hubs_create]
+
+## Building all the client libraries
+
+Open a command prompt/terminal:
+1. Execute: `git clone https://github.com/Azure/azure-sdk-for-java.git`
+1. Traverse to the repository root.
+1. Install the tooling and build the product by executing:
+    * `mvn install -Dinclude-non-shipping-modules -DskipTests -Dgpg.skip -f pom.client.xml`
```
Comment (JonathanGiles) [inline eventhubs/client/CONTRIBUTING.md]: This doesn't build anything - it just does the install

Comment (conniey) [reply eventhubs/client/CONTRIBUTING.md]: That's interesting. I always see it running tests, checkstyle, and stuff when I execute `mvn install`.
Is adding `mvn compile` prior to this better?
https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Built-in_Lifecycle_Bindings

Comment (conniey) [reply eventhubs/client/CONTRIBUTING.md]: Oh nevermind. it's becsause I always run `mvn clean install...`

**Pull request merged**